### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by Agroang**
For an internal project we have been implementing the use of `pages` and while doing so we faced some challenges. 
For example the use of Composition & Options API, the initial setup, among others.

This PR just updates the README file to try to make it easier if you have never used the gem.

### CODE REVIEW

**Comment by degicat** [comment]
@Resonious can you help us review this PR, please?

File: README.md [hunk 1]
```diff
@@ -46,20 +46,60 @@ The frontend portion is just a regular VueJS component.
...
 </script>
 ```
 
 When the page is loaded, the `data` block from `page.rb` is sent to `page.vue` automatically.
 
 ### Page actions
+Rails pages supports by default `GET` and `POST` requests. The following is an example of how you could do it:
+
+```
+# app/pages/my_page/page.rb
```
Comment (Resonious) [inline README.md:74]: Syntax highlighting :wink: 
```suggestion
```ruby
# app/pages/my_page/page.rb
```

File: README.md [hunk 2]
```diff
@@ -46,20 +46,60 @@ The frontend portion is just a regular VueJS component.
...
+  else
+    render json: {
+      error: 'Insufficient DogicaCoin :sad'
+    }
+  end
+end
+```
+
+```
+# app/pages/my_page/page.vue
```
Comment (Resonious) [inline README.md:90]:
```suggestion
```javascript
// app/pages/my_page/page.vue
```

**Review by Resonious — CHANGES_REQUESTED** [review]
Thanks! These are great additions. I'd just like to make the code examples look nicer with syntax highlighting.

**Pull request merged**